lib/servicelog/middlewares/job_headers.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'request_store'
+
+module Servicelog
+  class JobHeaders
```

Comment:
Can the middleware name be `ShoryukenStoreHeaders` to specify that it can only be used for Shoryuken gem?

Reply:
Changed